src/encoded/tests/test_audit_experiment.py

```diff
@@ -1272,6 +1290,52 @@ def test_audit_experiment_with_RNA_library_array_size_range(testapp, base_experi
                for error in collect_audit_errors(res))
 
 
+def test_audit_experiment_with_long_read_RNA_no_size_range(testapp, base_experiment,
+                                                            base_replicate,
+                                                            base_library):
+    testapp.patch_json(base_library['@id'], {'nucleic_acid_term_name': 'RNA'})
+    testapp.patch_json(base_replicate['@id'], {'library': base_library['@id']})
```

Comment:
I think the setup of this test is not imitating real experiment, hence the audit for long read RNA is not triggered, I think if you were to substitute the long read RNA by just "RNA-seq" it would still pass. To make it really work you should do something like you did in the "test_audit_experiment_with_RNA_library_missing_read_length" test. That brings me to the following point: you will probably need to create a fixture for the experiment you want to test, and in the test body use this fixture, potentially only switching the pipeline title.